rust_snuba/src/runtime_config.rs

```diff
@@ -48,4 +50,39 @@ mod tests {
         let config = get_str_config("test");
         assert_eq!(config.unwrap(), None);
     }
+
+    #[test]
+    fn test_runtime_config_is_stored_in_cache() {
+        crate::testutils::initialize_python();
+        let config = get_str_config("test");
+        assert_eq!(config.unwrap(), None);
+        assert!(CONFIG.read().contains_key("test"));
+    }
+
+    #[test]
+    fn test_runtime_config_retrieves_from_cache_within_deadline() {
+        CONFIG.write().insert(
+            "key".to_string(),
+            (
+                Some("value".to_string()),
+                Deadline::new(Duration::from_secs(10)),
+            ),
+        );
+        let config = get_str_config("key");
+        assert_eq!(config.unwrap(), Some("value".to_string()));
+        CONFIG.write().clear();
+    }
+
+    #[test]
+    fn test_runtime_config_invalidates_cache_outside_deadline() {
+        CONFIG.write().insert(
+            "key".to_string(),
+            (
+                Some("value".to_string()),
+                Deadline::new(Duration::from_secs(0)),
```

Comment:
I wanted to do Deadline::new(Duration::from_secs(1)) here, and then do a sleep but because we're using coarse time, the cache doesn't seem like it's invalidated?